composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/EmptyScreenContent.kt

```diff
@@ -5,9 +5,11 @@ import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import com.jetbrains.kmpapp.MR
-import dev.icerock.moko.resources.compose.stringResource
+import kmp_app_template.composeapp.generated.resources.Res
```

Comment:
@zsmb13 
According to [this](https://github.com/JetBrains/compose-multiplatform/blob/master/CHANGELOG.md#breaking-changes), you also need to add:
```kotlin
import kmp_app_template.composeapp.generated.resources.*
```